h/static/scripts/group-forms/components/GroupModeration.tsx

```diff
@@ -72,8 +73,9 @@ function AnnotationList({ filterStatus, classes }: AnnotationListProps) {
         filterStatus={filterStatus}
         loading={loading}
         annotations={annotations}
-        onAnnotationStatusChange={updateAnnotationStatus}
         removedAnnotations={removedAnnotations}
+        onAnnotationStatusChange={updateAnnotationStatus}
+        onAnnotationReloaded={updateAnnotation}
```

Comment:
If we added a refresh button or similar to the form in future we could run into an issue where there are conflicting updates. It might be better to centralize updates in one place.

Reply:
I'm not sure if I follow 🤔